src/ShareButton.scss

```diff
@@ -1,6 +1,6 @@
 @import '_variables';
 
-button.btn.ShareButton {
```

Comment:
We still need button.btn part to apply the default interaction styles to the light theme. With this removed,the  light theme has no hover style.

Reply:
Good catch, and fixed.